docs/low-code-sql/development/gems/datasources.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Data Sources
+id: datasources-sql
+description: access your data
+sidebar_position: 1
+tags:
+  - SQL
+  - sources
+  - seeds
+---
+
+Constitutes the set of Gems that help with loading data.
+
+## Source
+
+Read a table or view from the SQL Warehouse(s) specified in the Fabric.
+
+## Seed
```

Comment:
In [this doc](https://docs.prophecy.io/getting-started/sql-with-databricks#431-create-seeds), we say the seed is loaded to the warehouse as a table. Is that correct? It seems we store the file on git rather than the warehouse @several27 ?

Reply:
done